ROADMAP.md

```diff
@@ -0,0 +1,32 @@
+# Roadmap
+
+- [Raw devices Implement](#Raw-devices-Implement)
+- [Manage volume group that user created](#Manage-volume-group-that-user-created)
+- [Data Migration and Backup](#Data-Migration-and-Backup)
+- [Nodes and devices selector](#Node-and-devices-selector)
+
+###Raw devices Implement
+
+Not only lvm, operator could orchestrate csi driver that support raw devices. we should implement csi driver.
+
+###Manage volume group that user created
+
+volume group that created by user has available storage capacity, use could config operator that could manage these volume groups.
```

Comment:
can you please elaborate a bit more about how we plan to manage the volume groups?

Reply:
Users could config volume group that created by themself in CR (TopolvmCluster) instead of operator do that.